decidim-core/app/views/decidim/account/show.html.erb

```diff
@@ -11,6 +11,14 @@
     <div class="columns large-8 end">
       <%= form_required_explanation %>
 
+      <%= f.collection_select(
```

Comment:
Can we move this after the about text-area 🙏🏽 ? Thanks

Reply:
@andreslucena I have performed the requested change.

Reply:
Awesome 🚀